src/renderer/components/ft-toast/ft-toast.css

```diff
@@ -0,0 +1,35 @@
+.toast-holder {
```

Comment:
If you have multiple toast messages coming in and if one of those messages are much larger than the rest.  The rest of messages will end up matching the width of the largest message.  I'll let you make your best judgement on this if it ends up looking weird, however I'd be interested in seeing it where each toast message matches the width of the content within them.  Having them max at the current width that you have it is good though.  If this ends up looking too strange then we can leave it how it is.

Reply:
I've changed the styling so the width matches the content rather than the other toasts.